### DISCUSSION TITLE
Stackblitz live demo is now available

### DISCUSSION THREAD

**Discussion by ghiscoding** [6pac/SlickGrid#1077] [Show and tell]
### Stackblitz

You can also see this Vite demo working live in Stackblitz

[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/6pac/SlickGrid/tree/master/vite-demo)

**Comment by ghiscoding** [comment]
@6pac could you change the default project link on the project settings to point to that new Stackblitz? 
Here's the link to use:
https://stackblitz.com/github/6pac/SlickGrid/tree/master/vite-demo

**Comment by 6pac** [comment]
Sorry, I’ve had a look, but can’t find the setting you are referring to. Can you give me a bit more info?
B

From: Ghislain B. ***@***.***>
Date: Wednesday, 30 October 2024 at 11:49 AM
To: 6pac/SlickGrid ***@***.***>
Cc: Ben McIntyre ***@***.***>, Mention ***@***.***>
Subject: Re: [6pac/SlickGrid] Stackblitz live demo is now available (Discussion #1077)

@6pac<https://github.com/6pac> could you change the default project link on the project settings to point to that new Stackblitz?
Here's the link to use:
https://stackblitz.com/github/6pac/SlickGrid/tree/master/vite-demo

**Reply by ghiscoding** [reply]
yeah sure, I mean this link if you could change it to point to the Stackblitz instead of the Wikis

![image](https://github.com/user-attachments/assets/169bb38d-846b-4198-9e3f-83535a83d9e3)

**Comment by 6pac** [comment]
OK, done. I wasn’t sure about changing that one at first, but you can just scroll down ;-)

From: Ghislain B. ***@***.***>
Date: Wednesday, 30 October 2024 at 11:49 AM
To: 6pac/SlickGrid ***@***.***>
Cc: Ben McIntyre ***@***.***>, Mention ***@***.***>
Subject: Re: [6pac/SlickGrid] Stackblitz live demo is now available (Discussion #1077)

@6pac<https://github.com/6pac> could you change the default project link on the project settings to point to that new Stackblitz?
Here's the link to use:
https://stackblitz.com/github/6pac/SlickGrid/tree/master/vite-demo
